scripts/update/Updater.php

```diff
@@ -1344,5 +1344,9 @@ public function update($initialVersion)
         }
 
         $this->skip('42.0.4', '42.0.5');
+
+        if ($this->isVersion('42.0.5')) {
```

Comment:
you don't need to do this ,  just increase the skipped version 
```suggestion
      $this->skip('42.0.4', '42.1.0');
```